spec/examples/complete/Dockerfile

```diff
@@ -1,5 +1,8 @@
-FROM alpine
+FROM bash
```

Comment:
Didn't know about the bash container... thats nice to know about. Only slightly concerned as this may not be available in Container Yard... we should check.